shell/src/main/java/org/jtaccuino/jshell/extensions/dependency/Dependencies.java

```diff
@@ -92,33 +94,51 @@ static List<Path> resolve(String mavenCoordinates) {
 
             var dv = new DependencyVisitor() {
 
-                int depth = 0;
+                List<DependencyArtifact> deps = new ArrayList<>();
 
                 @Override
                 public boolean visitEnter(DependencyNode node) {
-                    System.out.println(" ".repeat(depth)
-                            + ":" + node.getDependency().getScope()
-                            + ":" + node.getDependency().getOptional()
-                            + ":" + node.getDependency().getArtifact()
-                            + " - " + node.getArtifact().getGroupId()
-                            + ":" + node.getArtifact().getArtifactId()
-                            + " Deps: " + node.getChildren().size());
-                    depth++;
+                    org.jtaccuino.jshell.extensions.dependency.DependencyArtifact newDep = new DependencyArtifact(
```

Comment:
Do you need the FQN class name?

```suggestion
                    DependencyArtifact newDep = new DependencyArtifact(
```